client/src/pages/BMIHistory.jsx

```diff
@@ -92,8 +90,15 @@ export default function BMIHistory() {
           ))}
         </ul>
       ) : (
-        <p>No BMI entries found.</p>
+        <div>
+        <h3 className="history-text">No bmi data yet...</h3>
+        <Link to="/health">
```

Comment:
Thank you for the improving the frontend when there's no BMI entries! Now it's aligns with yours if there's no Fitness entries. :)